test-utils/src/lib.rs

```diff
@@ -0,0 +1,102 @@
+//! A number of utilities useful for testing that we want to use in multiple crates
+
+use std::sync::Arc;
+
+use arrow_array::{ArrayRef, Int32Array, RecordBatch, StringArray};
+use arrow_schema::ArrowError;
+use delta_kernel::engine::arrow_data::ArrowEngineData;
+use delta_kernel::EngineData;
+use itertools::Itertools;
+use object_store::{path::Path, ObjectStore};
+use parquet::arrow::arrow_writer::ArrowWriter;
+use parquet::file::properties::WriterProperties;
+
+/// A common useful initial metadata and protocol. Also includes a single commitInfo
+pub const METADATA: &str = r#"{"commitInfo":{"timestamp":1587968586154,"operation":"WRITE","operationParameters":{"mode":"ErrorIfExists","partitionBy":"[]"},"isBlindAppend":true}}
+{"protocol":{"minReaderVersion":1,"minWriterVersion":2}}
```

Comment:
Would it be possible to make this a `Metadata` struct that we convert to string? I feel like that's easier to read.

Reply:
Yeah, there's lots of cleanup we can do. I'm preferring to just mostly move the code to its own crate (and a couple of minor cleanups) for now, and we can take on making this "better" in the future.

Reply:
sounds good! 👍